bleak/__init__.py

```diff
@@ -576,7 +576,13 @@ def services(self) -> BleakGATTServiceCollection:
         Gets the collection of GATT services available on the device.
 
         The returned value is only valid as long as the device is connected.
+
+        Raises:
+            BleakError: if :meth:`get_services` has not been called yet after establishing the connection.
```

Comment:
`get_services` is not a method on `BleakClient` so this link won't work (and really it is just an implementation detail of the backend), so we could probably just make a more generic comment here.